app/src/main/java/com/infomaniak/swisstransfer/ui/theme/Theme.kt

```diff
@@ -62,4 +82,7 @@ object SwissTransferTheme {
     val typography: CustomTypography
         @Composable
         get() = LocalCustomTypography.current
+    val colors: CustomColors
+        @Composable
+        get() = LocalCustomColors.current ?: error("Cannot access uninitialized LocalCustomColors")
```

Comment:
Avoid runtime exceptions

Reply:
Fixed by using the default unspecified value for the CustomColors class